gridstatus/ieso.py

```diff
@@ -822,7 +822,7 @@ def _request(self, url, verbose):
         sleep = 5
 
         while retry_num < max_retries:
-            r = requests.get(url)
+            r = requests.get(url, verify=False)
```

Comment:
The problem started on June 4 when the site updated its certificate. This solution is not acceptable for a production environment so I'm working through other options.

<img width="465" alt="Screenshot 2024-06-26 at 15 14 54" src="https://github.com/kmax12/gridstatus/assets/17484616/ab70a87a-f7b2-42c1-bb73-765533f4ae98">